### ISSUE TITLE
Great Expectations CLI: The command `suite` does not exist when used with the --v3-api flag.

### ISSUE THREAD

**Issue by phtechwohletz** [fivetran/great_expectations#2719]
**Describe the bug**
When attempting to use the CLI to initiate or edit an expectation suite from the V3 API, the command fails with the prompt that 'suite' does not exist.

``` {bash}
ec2-user$ great_expectations --v3-api suite
The command `suite` does not exist.
Please use one of: ['checkpoint', 'datasource', 'docs', 'init', 'project', 'store', 'suite', 'validation_operator']
Usage: great_expectations [OPTIONS] COMMAND [ARGS]...
Try 'great_expectations --help' for help.
Error: No such command 'suite'.
```

**To Reproduce**
1. Create a new data context using ```great_expectations --v3-api init```
2. Next attempt to create or utilize the expectation suite command. ```great_expectations --v3-api suite new```
**Note**: Doesn't work for ```great_expectations --v3-api suite --help``` or ```great_expectations --v3-api suite edit```
**Note**: ```great_expectations suite``` without the --v3-api flag works just fine but only uses the V2 API.

**Expected behavior**
The command executes and I am able to create a new expectation suite or at least see the available options for that command from the CLI.

**Environment (please complete the following information):**
 - Operating System: Amazon Linux 2, Python 3.7.7
 - Great Expectations Version: 0.13.18

**Additional context**
Also attempted using Python 3.8.8 and a completely fresh install, no change in behavior.

**Comment by Aylr** [comment]
Thank you for the excellent bug report @phtechwohletz ! With your nice report reproducing this was easy. I am working on a fix right now. It looks like a packaging issue (which are notoriously hard to catch) that was not exposed by our CI and manual testing. We hope to have a working release out Monday at the latest.

Thank you for catching and reporting this so quickly!!!

**Comment by phtechwohletz** [comment]
Thanks for your help!

**Comment by Aylr** [comment]
Fixed in v0.13.19
